content/manager/bootstrapping.md

```diff
@@ -165,6 +165,97 @@ Services:
+    ```
+
+
+## Command Line
+
+ 1. Switch to the new manager:
+
+    ```bash
+    $ export CLOUDIFY_USERNAME=cloudify
+    $ export CLOUDIFY_PASSWORD=cloudify
+    $ export CLOUDIFY_SSL_TRUST_ALL=true
+    $ cfy use --port 443 --management-ip ${CLOUDIFY_HOST}
+    ```
+
+    {{% gsNote title="TLS" %}}
+    If you have disabled security then use port 80 instead (in this case the `export` lines above are not required): `$ cfy use --port 80 --management-ip ${CLOUDIFY_HOST}`.
+
+    {{% /gsNote %}}
+
+ 2. Create an input file for the `CloudifySettings` blueprint
+    ```bash
+    $ echo >install.yaml <<EOF
+    {
+      "aws_access_key": "{your access key}",
+      "aws_secret_key": "{your secret key}",
+      "agents_security_group_name": "{your desired group}",
+      "user_ssh_key": "{your ssh key}",
+      "agents_user": "centos",
+      "agents_keypair_name": "{your desired keypair name}",
+      "new_manager_username": "cloudify",
+      "new_broker_password": "a bad password",
+      "broker_names_and_ips": "localhost",
+      "new_broker_username": "cloudify",
+      "manager_names_and_ips": "${CLOUDIFY_HOST}",
+      "new_manager_password": "a really bad password"
+    }
+    EOF
+    ```
+ 3. Create the deployment
+    ```bash
+    $ cfy deployments create --deployment-id install --blueprint-id CloudifySettings --inputs install.yaml
+    ```
+ 4. Start the `install` workflow
+    ```bash
+    $ cfy executions start --deployment-id install --workflow install
+    ```
+
+ 5. Save the TLS certificate for use with `cfy` in future
+
+    ```bash
+    # Stop trusting all:
+    $ unset CLOUDIFY_SSL_TRUST_ALL
+
+    $ export CLOUDIFY_SSL_CERT=${CLOUDIFY_HOST}.cert
+    # hack to stop openssl getting stuck | download the certificate     | pipe it to sed to grab just the certificate from the output             >pipe to file
+    $ echo | openssl s_client -connect ${CLOUDIFY_HOST}:443 2>/dev/null | sed -n '/^-----BEGIN CERTIFICATE-----$/,/^-----END CERTIFICATE-----$/p' >${CLOUDIFY_SSL_CERT}
+    ```
+
+
+## Web UI
+
+ 1. In your browser, navigate to http://{your-manager-ip}
+    {{% gsNote %}}
+    If security is enabled, your browser will complain that the TLS/SSL certificate is not valid. Temporarily allow the connection: A new certificate will be generated as part of this setup process.
+    {{% /gsNote %}}
+ 2. If asked, authenticate using "cloudify" as both the username and password.
+ 3. Create a new deployment of the `CloudifySettings` blueprint:
+    ![CloudifySettings blueprint deploy]({{</* img "manager/image-deploy-new-deployment.png" */>}})
+ 4. Once the new deployment is ready, execute the `install` workflow:
+    TODO: add image
+ 5. Fill in the required fields:
+    ![CloudifySettings inputs]({{</* img "installation/image-deploy-deployment-inputs.png" */>}})
+    {{% gsNote %}}
+    It is possible to use the contents of an inputs file (if formatted as JSON) directly by pasting them into the "raw" field.
+    {{% /gsNote %}}
+
+
```

Comment:
please also specify what the prerequisites are in the `prerequisites` page for using images (like opening ports, creating networks, security groups, whatever...)